src/repositories/implementations/accountRepository.ts

```diff
@@ -12,31 +12,62 @@ export default class AccountRepository implements IAccountRepository {
    */
   private db: Db;
 
+  /**
+   * Accounts persistent storage collection
+   */
+  private collection: Collection;
+
   /**
    * Creates account repository instance
    *
    * @param db - MongoDB client
    */
   constructor(db: Db) {
     this.db = db;
+    this.collection = this.db.collection('accounts');
   }
 
   /**
-   * Fetches MongoDB to get AccountData and returns Account Model
+   * Fetches MongoDB to get AccountConstructorData and returns Account Model
    *
    * @param id - account identifier
    */
-  public getAccount(id: string): Account {
-    // @todo Here we fetch MongoDB to get account data and dr/cr amount
-    // const account = await this.db.collection('account').findOne({ id: id });
-
-    return new Account({
+  public async getAccount(id: string): Promise<Account|null> {
```

Comment:
в названиях других методов нет слова Account, так что и тут не хотелось бы. Если просто `get` нельзя использовать то можно `find`
 
```suggestion
  public async find(id: string): Promise<Account|null> {
```

Reply:
done